application/org.openjdk.jmc.rjmx/META-INF/MANIFEST.MF

```diff
@@ -43,10 +43,9 @@ Export-Package: org.openjdk.jmc.rjmx,
    org.openjdk.jmc.console.uitest,
    org.openjdk.jmc.flightrecorder.controlpanel.ui,
    org.openjdk.jmc.rjmx.ui,
-   org.openjdk.jmc.rjmx.services.jfr,
    org.openjdk.jmc.rjmx.services.jfr.test,
    org.openjdk.jmc.rjmx.test",
- org.openjdk.jmc.rjmx.services.internal;x-friends:="org.openjdk.jmc.rjmx.services.jfr,org.openjdk.jmc.rjmx.test",
+ org.openjdk.jmc.rjmx.services.internal;x-friends:="org.openjdk.jmc.rjmx.test",
```

Comment:
Line 55 ` org.openjdk.jmc.rjmx.subscription.storage.internal;x-internal:=true,` has to be removed, otherwise I see an error in Eclipse.